css/style.css

```diff
@@ -16,4 +17,175 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+.container {
+    margin: 0 auto;
+}
+
+h2, h3 {
+    font-weight: 300;
+
+}
+/*Header*/
+
+ header {
+    display: flex;  
+}
+
+.logo {
+    width: 2em;
+    padding-left: 2em;
+}
+
+nav {
+    display: flex;
+    padding: 1em 0;
+}
+
+ul {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+}
+
+li {
+    padding: 0 1em;
+    cursor: pointer;
+}
+
+li:first-child {
+    font-weight: bold;
+}
+
+li:hover {
+    color:#F15B2A;
+}
+
+/*Jumbotron*/
+
+h1 {
+    font-weight: 300;
+    font-size: 2.5em;
+    opacity: 0.9;
+}
+
+ .jumbotron {
+     display: flex;
+     flex-direction: column;
+     align-items: center;
+     justify-content: space-around;
+     background-image: url("../img/first-background.jpg");
+     background-position: center;
+     background-size: cover;
+     padding: 50px 40px;
+     color: #fff;    
+ }
+
+ .jumbotron p {
+     font-weight: 100;
+     font-size: 120%;
+     opacity: 0.9;
+ }
+
+ .jumbotron button {
+     background-color: #F15B2A;
+     max-width: max-content;
+     border: none;
+     border-radius: 3px;
+     padding: 0.8em;
+ }
+ /*Main content*/
+
+ article {
+     padding: 2em 0;
+ }
+
+ 
+
+ .icons {
+     display: flex;
+     flex-direction: column;  
+     align-items: center;
+     padding:2em 0;
+ }
+ .icon {
+     padding: 1.5em 0;
+ }
+
+ .icon img {
+     max-width: 6em;
+ }
+
+.icon-title {
+    font-size: 120%;
+}
+
+ /*footer*/
+
+ footer {
+     display: flex;
+     flex-direction: column;
+     align-items: center;
+ }
```

Comment:
```suggestion
```
The elements are already laid out vertically, and they're already centred, so we don't need to add this